2024/YoctoProjectDeveloperDay/practical-sec-tee.md

```diff
@@ -0,0 +1,994 @@
+(...)
+root@quartz64-a:~# xtest
+(...)
+653 subtests of which 188 failed
+106 test cases of which 76 failed
+0 test cases were skipped
+TEE test application done!
+```
+
+.center[A hard to debug problem without source code!]
+
+???
+
+- Time for this slide: TODO
+- Idea/goal of this slide: TODO
+- What to say:
+    - TODO
+- Notes:
+    - TODO
+
+---
+
+# Testing self-ported image
+
+```bash
+U-Boot SPL 2024.01 (Jan 08 2024 - 15:37:48 +0000)
+(...)
+NOTICE:  BL31: v2.3():v2.3-607-gbf602aff1:cl
+NOTICE:  BL31: Built : 10:16:03, Jun  5 2023
+(...)
+I/TC: OP-TEE version: 4.1.0-dev (gcc version 13.2.0 (GCC)) #1 Fri Jan 19 17:14:14 UTC 2024 aarch64
+(...)
+root@quartz64-a:~# alias p11="pkcs11-tool --module /usr/lib/libckteec.so.0"
+root@quartz64-a:~# p11 --list-slots
+Available slots:
+Slot 0 (0x0): OP-TEE PKCS11 TA - TEE UUID 94e9ab89-4c43-56ea-8b35-45dc07226830
+  token state:   uninitialized
+(...)
+root@quartz64-a:~# p11 --init-token --label mytoken --so-pin 1234567890
+(...)
+root@quartz64-a:~# p11 --list-slots
+Available slots:
+Slot 0 (0x0): OP-TEE PKCS11 TA - TEE UUID 94e9ab89-4c43-56ea-8b35-45dc07226830
+  token label        : mytoken
+  token manufacturer : Linaro
+(...)
+root@quartz64-a:~# xtest
+(...)
++-----------------------------------------------------
+28130 subtests of which 0 failed
+106 test cases of which 0 failed
+0 test cases were skipped
+TEE test application done!
+```
+
+???
+
+- Time for this slide: TODO
+- Idea/goal of this slide: TODO
+- What to say:
+    - TODO
+- Notes:
+    - TODO
+
+---
+
+<!-- Zarhus and Zarhus OS -->
+
+# Contact us
+
+.left-column45[ <img src="/img/zarhus_logo.png" height="150px"
+style="margin-top:-10px; margin-left:200px"> ]
+
+.right-column55[
+
+## [Zarhus OS](https://docs.zarhus.com/)
+
+]
+
+<br>
+<br>
+<br>
+<br>
+<br>
+
+.right-column50[
+
+- <a href="https://3mdeb.com">https://3mdeb.com</a>
+
+- <a href="https://calendly.com/3mdeb/consulting-remote-meeting">Book a call</a>
+
+- <a href="https://newsletter.3mdeb.com/subscription/PW6XnCeK6">
+      Sign up for the newsletter
+    </a>
+  ]
+
+.left-column50[
+
+- <a href="https://www.linkedin.com/company/3mdeb">
+    <img src="/remark-templates/3mdeb-presentation-template/images/linkedin.png"
+      width="24px" style="margin-bottom:-5px; margin-left:-15px"/>
+      linkedin.com/company/3mdeb
+    </a>
+
+- <a href="mailto:contact@3mdeb.com">
+    <img src="/remark-templates/3mdeb-presentation-template/images/email.png"
+      width="24px" style="margin-bottom:-5px; margin-left:-15px"/>
+      contact@3mdeb.com
+  </a>
+
+- <a href="https://www.facebook.com/3mdeb">
+    <img src="/remark-templates/3mdeb-presentation-template/images/facebook.png"
+      width="24px" style="margin-bottom:-5px; margin-left:-15px"/>
+      facebook.com/3mdeb
+  </a>
+
+- <a href="https://twitter.com/3mdeb_com">
+    <img src="/remark-templates/3mdeb-presentation-template/images/twitter.png"
```

Comment:
@DaniilKl outdated icon - if you've some time, maybe prepare the X icon - nice to have feature :)

Reply:
Fixed here 75b8fe13c6e780899601554d87acea371b6df506.